syntax = "proto3";

package fbsrankings.messages.query;

message GameCountBySeasonResult {
    string query_id = 1;
    string season_id = 2;
    int32 count = 3;
}

message GameCountBySeasonQuery {
    string query_id = 1;
    string season_id = 2;
}
